cookbooks/cdo-nginx/test/cookbooks/nginx_test/templates/default/init.d.erb

```diff
@@ -0,0 +1,88 @@
+#!/bin/sh
+set -e
+# Example init script, this can be used with nginx, too,
+# since nginx and unicorn accept the same signals
+
+# Feel free to change any of the following variables for your app:
+TIMEOUT=${TIMEOUT-60}
+APP_ROOT=<%= @app_root %>
+PID=<%= @pid_file %>
+CMD="cd $APP_ROOT; LANG=en_US.UTF-8 bundle exec unicorn -D -c <%= @src_file %> -E <%= @env %>"
```

Comment:
Just to confirm:  This CMD ties it to unicorn rather than nginx, so the comment above should perhaps be clarified.